src/Components/Navigation.js

```diff
@@ -153,4 +165,14 @@ class Navigation extends React.Component {
   }
 }
 
+function BackgroundVideo({ videoUrl }) {
```

Comment:
Can you move `BackgroundVideo` below `ActualNavigation`, but above `Navigation`?